dynamiqs/smesolve/smesolve.py

```diff
@@ -217,18 +217,18 @@ def smesolve(
         )
 
     # format and batch all tensors
-    # H: (b_H, 1, 1, n, n)
-    # rho0: (b_H, b_rho0, ntrajs, n, n)
+    # H: (b_H, 1, 1, 1, n, n)
+    # rho0: (b_H, b_jump_ops, b_rho0, ntrajs, n, n)
     # exp_ops: (len(exp_ops), n, n)
-    # jump_ops: (len(jump_ops), n, n)
+    # jump_ops: (len(jump_ops), b_jump_ops, n, n)
     H = to_td_tensor(H, dtype=options.cdtype, device=options.device)
     rho0 = to_tensor(rho0, dtype=options.cdtype, device=options.device)
     H = batch_H(H).unsqueeze(2)
-    rho0 = batch_y0(rho0, H).unsqueeze(2).repeat(1, 1, ntrajs, 1, 1)
+    jump_ops = batch_jump_ops(jump_ops, dtype=options.cdtype, device=options.device)
```

Comment:
Don't you need to unsqueeze this guy too? Did you test that the SME solver still works? I can provide a code example if needed.

Reply:
Yes an example would be good please 🙏